roles/autotested/tasks/main.yml

```diff
@@ -32,6 +32,11 @@
     path: "{{ autotest_dest_dir }}/RELEASE-VERSION"
     state: absent
 
+- name: Bundled Docker Autotest directory is removed
+  file:
+    path: "{{ autotest_docker_path }}"
+    state: absent
+
```

Comment:
FWIW here's my playbook

     # git repos: autotest and autotest-docker. update=no means "chill": it
     # lets us modify files in the repos without Ansible barfing on future
     # runs. If you ever want to update, just rm -rf the checkout manually.
     - name: install autotest
        git: repo=https://github.com/autotest/autotest.git
            dest={{ destdir }} update=no recursive=no

     - name: install autotest-docker
       git: repo=https://github.com/autotest/autotest-docker.git
            dest="{{ destdir }}/client/tests/docker" update=no